docs/package.json

```diff
@@ -16,8 +16,7 @@
     "docs:scripts:generate-legacy-docs-versions": "node .vuepress/tools/generate-legacy-docs-versions.mjs",
     "docs:start": "npm run docs:scripts:link-assets && node --experimental-fetch ./node_modules/.bin/vuepress dev ./",
     "docs:start:no-cache": "npm run docs:scripts:link-assets && node --experimental-fetch ./node_modules/.bin/vuepress dev --no-cache ./",
-    "docs:build": "node .vuepress/tools/build.mjs",
-    "docs:build:no-cache": "npm run docs:build -- --no-cache",
```

Comment:
Thank you so much, I wished for that for a long time.

Can you also remove the `no-cache` option from `start`? On our Slack you will find many stories where cache was troublesome there as well.

Also, @jansiegel can you check if this problem still exists? https://handsoncode.slack.com/archives/C034WPQUG9W/p1658917802885339?thread_ts=1658914565.788989&cid=C034WPQUG9W